setup.py

```diff
@@ -59,10 +59,12 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Framework :: Django',
         'Framework :: Django :: 1.8',
         'Framework :: Django :: 1.9',
```

Comment:
Django 1.9 isn't officially supported, should we remove it from the classifiers?

Reply:
Sharp 😎